buildroot-external/overlay/base-raspmatic/bin/checkAddonUpdates.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 # shellcheck shell=dash disable=SC3010
 
+# only continue if not disabled
+[[ -e /etc/config/NoCheckAddonUpdates ]] && exit 1
```

Comment:
Statt das in der `checkAddonUpdates.sh` zu machen würde ich das wohl besser in der standard crontab analog zu den anderen Einträgen erledigen.

Reply:
[b158264](https://github.com/jens-maus/RaspberryMatic/pull/2069/commits/b1582647784545c7e017e5d0226d8e79ee8c85fc)